source/includes/code-examples/FindManyAsync.cs

```diff
@@ -0,0 +1,97 @@
+using MongoDB.Bson;
+using MongoDB.Bson.Serialization.Attributes;
+using MongoDB.Bson.Serialization.Conventions;
+using MongoDB.Driver;
+using MongoDB.Driver.Linq;
+using static System.Console;
+
+namespace CSharpExamples.UsageExamples;
+
+public class FindManyAsync
+{
+    private static IMongoCollection<Restaurant> _restaurantsCollection;
+    private static string _mongoConnectionString = "<Your MongoDB URI>";
+
+    public static void Main(string[] args)
+    {
+        Setup();
+
+        // Find multiple documents using builders
+        WriteLine("Finding documents with builders...:");
+        var restaurantsBuilders = FindMultipleRestaurantsBuilderAsync();
+        WriteLine("Number of documents found: " + restaurantsBuilders.Result.Count);
+
+        // Extra space for console readability 
+        WriteLine();
+
+        // Find multiple documents using LINQ
+        WriteLine("Finding documents with LINQ...:");
+        var restaurantsLINQ = FindMultipleRestaurantsLINQAsync();
+        WriteLine("Number of documents found: " + restaurantsLINQ.Result.Count);
+
+        WriteLine();
+
+        // Find all documents
+        WriteLine("Finding all documents...:");
+        var allRestaurants = FindAllRestaurantsAsync();
+        WriteLine("Number of documents found: " + allRestaurants.Result.Count);
+    }
+
+    private static async Task<List<Restaurant>> FindMultipleRestaurantsBuilderAsync()
+    {
+        // start-find-builders-async
```

Comment:
What does `Builders` mean in this method name?

Reply:
It's meant to differentiate it from the LINQ version of the method.